lib/referencer.js

```diff
@@ -593,15 +593,26 @@ class Referencer extends esrecurse.Visitor {
         this.visitChildren(node);
     }
 
+    // TODO: ExportDeclaration doesn't exist. for bc?
     ExportDeclaration(node) {
         this.visitExportDeclaration(node);
     }
 
+    ExportAllDeclaration(node) {
+        this.visitExportDeclaration(node);
```

Comment:
shouldn't this just be empty?
`ExportAllDeclaration` is specced to always have a source (eg it always exports from an external module and defines no local variables).

I know that `visitExportDeclaration` does `if (node.source) { do nothing }`, but why bother with the function call?

Reply:
Because I thought that people expect the method `visitExportDeclaration` to be called on all export declarations because of the name.